containers/go-postgres/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# [Choice] Go version (use -bullseye variants on local arm64/Apple Silicon): 1, 1.16, 1.17, 1-bullseye, 1.16-bullseye, 1.17-bullseye, 1-buster, 1.16-buster, 1.17-buster
+ARG VARIANT=1-bullseye
+FROM mcr.microsoft.com/vscode/devcontainers/go:0-${VARIANT}
+
+# [Choice] Node.js version: none, lts/*, 16, 14, 12, 10
+ARG NODE_VERSION="none"
+RUN if [ "${NODE_VERSION}" != "none" ]; then su vscode -c "umask 0002 && . /usr/local/share/nvm/nvm.sh && nvm install ${NODE_VERSION} 2>&1"; fi
+
+# [Optional] Uncomment this section to install additional OS packages.
+# RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+#     && apt-get -y install --no-install-recommends <your-package-list-here>
+
+# [Optional] Add more packages to the below lines to use go get to install anything else you need
+USER vscode
+RUN go get -x github.com/lib/pq
```

Comment:
Two things:
1. Since this ends up in the image, it might be better not to install this unless it provides some command line utilities people will want.  Is there a functional reason for it? I'd expect people would usually use a go.mod file with specific versions and then use that to "go get".  The other things we install are command line utilities like gopls and we actually clean out the library content to avoid conflicts.

2. Might be best to switch back to root afterwards.  The issue with defaulting to a non-root user is you end up having to use sudo for things like docker-in-docker. The pattern we use instead is to have the container run as root, but you connect with VS Code as non-root via the "remoteUser" property.

Reply:
pushed a fix, following up offline